deploy/lib/server_config.rb

```diff
@@ -1026,8 +1032,22 @@ def save_files_to_fs(target_db, target_dir)
           # create the directory so that it will exist when we try to save files
           Dir.mkdir("#{target_dir}" + uri)
         else
-          r = go("#{@protocol}://#{@hostname}:#{@bootstrap_port}/qconsole/endpoints/view.xqy?dbid=#{db_id}&uri=#{uri}", "get")
-          File.open("#{target_dir}#{uri}", 'wb') { |file| file.write(r.body) }
+          # check type, if REST use that endpoint, otherwise use a query
+          if (port != nil)
```

Comment:
I think this is unnecessary. The port value is not used, and the downloading of files depends on QConsole, which works for any app-type..